README.md

```diff
@@ -16,25 +16,24 @@ This is a **fork** of the MultiMC Launcher and not endorsed by MultiMC.
 
 There are development builds available [here](https://github.com/PrismLauncher/PrismLauncher/actions). These have debug information in the binaries, so their file sizes are relatively larger.
 
-Portable builds are provided for on Linux, Windows, and macOS.
+Portable builds are provided for Linux, Windows, and macOS.
 
-For Debian and Arch, you can use these packages for the latest development versions:  
-[![prismlauncher-git](https://img.shields.io/badge/aur-prismlauncher--git-blue)](https://aur.archlinux.org/packages/prismlauncher-git/)
-[![prismlauncher-git](https://img.shields.io/badge/mpr-prismlauncher--git-orange)](https://mpr.makedeb.org/packages/prismlauncher-git)  
-## Help & Support
+For Debian and Arch, you can use these packages to get compiled development versions:  
 
-Feel free to create an issue if you need help. However, you might find it easier to ask in the Discord server.
+[![prismlauncher-git](https://img.shields.io/badge/mpr-prismlauncher--git-orange?style=flat-square&logo=appveyor)](https://mpr.makedeb.org/packages/prismlauncher-git)  [![prismlauncher-git](https://img.shields.io/badge/aur-prismlauncher--git-blue?style=flat-square&logo=appveyor)](https://aur.archlinux.org/packages/prismlauncher-git/)
```

Comment:
link to https://prismlauncher.org/wiki/development/development-builds/ or add the MPR / Gentoo ones :D

Reply:
So I add these at the development build thing?

Reply:
![image](https://user-images.githubusercontent.com/66513643/197903994-b7092ae1-ab9d-4915-bbc3-4abc177ffe17.png)
Like this?

Reply:
:+1: